main/src/main/scala/org/clulab/odin/Mention.scala

```diff
@@ -140,10 +143,10 @@ trait Mention extends Equals with Ordered[Mention] with Serializable {
     case Some(txt) => txt.slice(startOffset, endOffset)
     case None =>
       // try to reconstruct the sentence using the character offsets
-      val bits = words.head +: tokenInterval.drop(1).map { i =>
+      val bits = raw.head +: tokenInterval.drop(1).map { i =>
```

Comment:
It occurs to me that the partner to head is tail.  Since we are assuming a head, tail should be available, so I wonder if tokenInterval.tail would be OK.  It might otherwise be take(1) and drop(1).

Reply:
this is rhetorical, right?

Reply:
No, actually.  It's certainly trivial, but I didn't want to change it without a heads up.